yt_dlp/extractor/toggle.py

```diff
@@ -12,217 +12,328 @@
 
 
 class ToggleIE(InfoExtractor):
-    IE_NAME = 'toggle'
-    _VALID_URL = r'(?:https?://(?:(?:www\.)?mewatch|video\.toggle)\.sg/(?:en|zh)/(?:[^/]+/){2,}|toggle:)(?P<id>[0-9]+)'
-    _TESTS = [{
-        'url': 'http://www.mewatch.sg/en/series/lion-moms-tif/trailers/lion-moms-premier/343115',
-        'info_dict': {
-            'id': '343115',
-            'ext': 'mp4',
-            'title': 'Lion Moms Premiere',
-            'description': 'md5:aea1149404bff4d7f7b6da11fafd8e6b',
-            'upload_date': '20150910',
-            'timestamp': 1441858274,
+    IE_NAME = "toggle"
+    _VALID_URL = r"(?:https?://(?:(?:www\.)?mewatch|video\.toggle)\.sg/(?:[^/]+/){1,}|toggle:)[^/]+-(?P<id>[0-9]+)"
+    _TESTS = [
+        # [test skipped]
```

Comment:
Please use the skip parameter, like in line 59 (prior to this PR)

I've noticed there's also a `skip_download` parameter, but I'm not entirely sure what the difference is between them. Either way, there's no need to comment out tests, especially existing tests that already have the `skip` parameter

Reply:
`'params': {'skip_download' True}` is the same as adding `--skip-download` to the CLI; it will still extract but not download. Use it when extraction should be tested but downloading shouldn't, either because the md5 hash will vary or the m3u8 fragment size is too small.

Use `skip` when the test URL can no longer be extracted